notify/mail.sh

```diff
@@ -78,8 +78,13 @@ mail_send() {
 
 _mail_bin() {
   _MAIL_BIN=""
+  _MAIL_BINS="sendmail ssmtp mutt mail msmtp"
 
-  for b in "$MAIL_BIN" sendmail ssmtp mutt mail msmtp; do
```

Comment:
why not just remove the double quotation marks here?

```
for b in $MAIL_BIN sendmail ssmtp mutt mail msmtp; do
```

Reply:
:facepalm: So true... it's so simple, yet I failed to see that. Thank you.